tests/tooling/test_generate_markdown_command_exports.py

```
@@ -0,0 +1,1074 @@
+
+            original_apply = self.script._apply_agent_output_plan
+
+            def fail_preview_first(
+                *,
+                rendered_outputs: list[tuple[Path, str]],
+                stale_paths: list[Path],
+                output_base: Path,
+            ) -> list[Path]:
+                del rendered_outputs, stale_paths
+                if "tooling/sync/generated/with-conditional/markdown" in str(output_base):
+                    raise CommandSurfaceLoadError("preview write failed")
```

Ruff (0.15.4)

[warning] Avoid specifying long messages outside the exception class

(TRY003)